src/api/products/ProductList.ts

```diff
@@ -14,14 +19,38 @@ export class ProductList extends BaseList<
   getPageInfo: GetPageInfo<ProductListQuery> = result =>
     result.data.products?.pageInfo!;
 
+  getTotalCount: GetTotalCount<ProductListQuery> = result =>
+    result.data.products?.totalCount!;
+
   mapQueryData: MapQueryData<
     ProductListQuery,
     ProductList_products_edges_node
   > = data => data.products?.edges.map(({ node }) => node);
 
-  query = (variables: ProductListVariables) =>
+  query = async (variables: ProductListVariables) =>
```

Comment:
looks like some leftover along with the comment below

Reply:
I was sure I removed this... 🙄  My bad!